docs/changelog.md

```diff
@@ -222,6 +244,10 @@ description: |
 
 - Syntax
   - Added support for nested imports like `{import "file.typ": module.item}`
+    <!-- Add docs for this in
```

Comment:
I added a bit of docs to scripting

Reply:
Sorry, I was looking in the wrong tab, I see it now.

![image](https://github.com/user-attachments/assets/c89ae1c6-07db-42e1-8f6f-e1bad47f2b37)